modules/gdnative/videodecoder/video_stream_gdnative.cpp

```diff
@@ -125,7 +125,12 @@ bool VideoStreamPlaybackGDNative::open_file(const String &p_file) {
 		texture_size = *(Vector2 *)&vec;
 
 		pcm = (float *)memalloc(num_channels * AUX_BUFFER_SIZE * sizeof(float));
-		memset(pcm, 0, num_channels * AUX_BUFFER_SIZE * sizeof(float));
+
+		//VideoStreamGDNativeAudioFix Branch
```

Comment:
You shouldn't put comments identifying the provenance of the code, that's what we use Git for :)

Reply:
@akien-mga i will fix it, sorry for that.